src/lazy/expanded/macro_evaluator.rs

```diff
@@ -1272,6 +1259,81 @@ mod tests {
         )
     }
 
+    #[test]
+    fn macros_close_over_dependencies() -> IonResult<()> {
```

Comment:
🗺️ This unit test demonstrates defining a macro with a dependency on an unaddressable macro.